lib/util/merge-confidence/index.ts

```diff
@@ -25,14 +25,45 @@ export const confidenceLevels: Record<MergeConfidence, number> = {
 export function initConfig(): void {
   apiBaseUrl = getApiBaseUrl();
   token = getApiToken();
+  supportedDatasources = [...presetSupportedDatasources];
+
   if (!is.nullOrUndefined(token)) {
     logger.debug(`Merge confidence token found for ${apiBaseUrl}`);
   }
 }
 
+export function parseSupportedDatasourceString(
```

Comment:
Should we move this to the preset side and make the preset datasources configure as well?